pytorch/src/main/java/org/bytedeco/pytorch/presets/torch_cuda.java

```diff
@@ -21,37 +21,40 @@
  */
 package org.bytedeco.pytorch.presets;
 
+import org.bytedeco.cuda.presets.cudnn;
+import org.bytedeco.cuda.presets.cusolver;
+import org.bytedeco.cuda.presets.cusparse;
 import org.bytedeco.javacpp.ClassProperties;
 import org.bytedeco.javacpp.LoadEnabled;
 import org.bytedeco.javacpp.annotation.*;
 import org.bytedeco.javacpp.tools.Info;
 import org.bytedeco.javacpp.tools.InfoMap;
 import org.bytedeco.javacpp.tools.InfoMapper;
+import org.bytedeco.pytorch.presets.torch.PointerInfo;
 
 /**
  * @author Hervé Guillemet
  */
 @Properties(
-    inherit = torch.class,
+    inherit = { torch.class, cudnn.class, cusparse.class, cusolver.class },
     value = {
         @Platform(
             extension = "-gpu",
+            // define = "USE_C10D_NCCL", // Not on Windows
             include = {
-                "ATen/cudnn/Descriptors.h",
                 "ATen/cudnn/Types.h",
-                "c10/cuda/CUDAGuard.h",
+                "ATen/cudnn/Descriptors.h",
+                "ATen/cuda/CUDAEvent.h",
                 "torch/csrc/inductor/aoti_runner/model_container_runner_cuda.h",
 
                 // For inclusion in JNI only, not parsed
                 "ATen/cuda/CUDAGeneratorImpl.h",
             },
-            link = { "cudart", "cusparse", "cudnn" },
-            linkpath = {
-                "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v12.3/lib/x64/",
-                "/usr/local/cuda-12.3/lib64/",
-                "/usr/local/cuda/lib64/",
-                "/usr/lib64/"
-            }
+            exclude = {
+                "<cublas.h>", // pytorch includes cublas_v2, which is not compatible with cublas included from inherited cudnn presets
+                "<driver_functions.h>" // causes #warning
+            },
+            library = "jnitorch"
```

Comment:
Please don't do that, leave it as jnitorch_cuda

Reply:
If I remember well, I had to do this because some callback `std::function` were used from headers parsed both in torch and torch_cuda, which results in Generator producing same glue code in torch JNI and torch_cuda JNI, and in a linking error.
But I just tried again without `library =` and it worked, so I guess I finally didn't map the offending function.